pkg/restapi/did/controller_test.go

```diff
@@ -38,10 +39,10 @@ type mockRuleProvider struct {
 	Err error
 }
 
-func (p *mockRuleProvider) Transform(uri string) (string, error) {
+func (p *mockRuleProvider) Resolve(uri string) (rules.ResolveResult, error) {
```

Comment:
I think it is a bit confusing for rule provider to have Resolve method (it will be confused with dids) - maybe Match makes more sense if you don't like original Transform.